libs/elements/layout/header/src/lib/components/header/header.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'elewa-website-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss'],
+})
+export class HeaderComponent implements OnInit {
+  ngOnInit(): void {
+    const hambuger = document.querySelector('.hambuger');
+    const navMenu = document.querySelector('.nav-menu');
+
+    hambuger?.addEventListener('click', () => {
+      hambuger.classList.toggle('active');
+      navMenu?.classList.toggle('active');
+    });
+
+    navMenu?.querySelectorAll('.nav-links').forEach((link) => {
+      console.log(link);
+      link.addEventListener('click', () => {
+        hambuger?.classList.remove('active');
+        navMenu.classList.remove('active');
+      });
+    });
+  }
```

Comment:
Use a custom Directive to manage and manipulate this style. Remember that direct DOM manipulation should be avoided whenever possible in Angular applications (any framework actually), as it can lead to unexpected behaviour, performance issues, and security vulnerabilities. Use ElementRef and Renderer2 when necessary in the directive you create